symfony/framework-bundle/4.4/config/packages/framework.yaml

```diff
@@ -0,0 +1,20 @@
+# see https://symfony.com/doc/current/reference/configuration/framework.html
+framework:
+    secret: '%env(APP_SECRET)%'
+    #csrf_protection: true
+    #http_method_override: true
+
+    secrets:
+        vault_directory: '%kernel.project_dir%/config/secrets/%env(APP_VAULT)%'
```

Comment:
What about making this the default in the [fwb config](https://github.com/symfony/symfony/blob/dfc32678c186f93f1c3ead8b02231aa3732d869b/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L147 ) (adding the `env(APP_VAULT): '%env(APP_ENV)%'` parameter as default in the fwb as well)?

Reply:
I'm not sure we should reference any env vars in Configuration files - we don't have any right now at least. We could add `kernel.vault` and make it default to APP_VAULT in `MicroKernelTrait` (which is the only place where we have some hardcoded env var reference.

For 4.4, the current is the only option thought.